### PR TITLE
Onboard HYPR authenticator

### PR DESCRIPTION
**Pull request by Yasasr1**
### Purpose
This PR adds the UI implementations required to onboard HYPR authenticator

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+} from "./hypr-authentication-provider-create-wizard";
+import { getIdentityProviderList } from "../../../api";
+import { IdentityProviderManagementConstants } from "../../../constants";
+import { IdentityProviderListResponseInterface, IdentityProviderTemplateInterface } from "../../../models";
+import { handleGetIDPListCallError } from "../../utils";
+
+/**
+ * Proptypes for the HyprAuthenticationWizardFrom.
+ */
+interface HyprAuthenticationProviderCreateWizardContentPropsInterface extends TestableComponentInterface {
```
Comment (pavinduLakshan) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx:34]: Let's use `IdentifiableComponentInterface` as `TestableComponentInterface` is deprecated.

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+    /**
+     * Trigger form submit.
+     * @param submitFunctionCb - Callback.
+     */
+    triggerSubmission: (submitFunctionCb: () => void) => void;
```
Comment (pavinduLakshan) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx:39]: The param name `submitFunctionCb` feels ambiguous IMO. Shall we rename as follows?

```suggestion
    triggerSubmission: (submitFunctionCallback: () => void) => void;
```

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+): ReactElement => {
+
+    const {
+        triggerSubmission,
+        triggerPrevious,
+        changePageNumber,
+        template,
+        setTotalPage,
+        onSubmit,
+        [ "data-testid" ]: testId
```
Comment (pavinduLakshan) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx:87]: Let's use `data-componentid` instead as `testid` is now deprecated.

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+     */
+    const idpNameValidation = (value: string): string => {
+
+        let nameExist: boolean = false;
+
+        if (idpList?.count > 0) {
+            idpList?.identityProviders.map((idp: IdentityProviderTemplateInterface) => {
+                if (idp?.name === value) {
+                    nameExist = true;
+
```
Comment (pavinduLakshan) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx:136]: Remove this additional new line.

File: apps/console/src/features/identity-providers/data/identity-provider-templates/templates/hypr/create-wizard-help.tsx [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { TestableComponentInterface } from "@wso2is/core/models";
```
Comment (pavinduLakshan) [inline apps/console/src/features/identity-providers/data/identity-provider-templates/templates/hypr/create-wizard-help.tsx:19]: Let's use `IdentifiableComponentInterface` instead

File: apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx [hunk 6]
```diff
@@ -251,7 +252,7 @@ export const AuthenticatorCreateWizardFactory: FunctionComponent<AuthenticatorCr
         getIdentityProviderList(null, null, "name sw " + idpName)
             .then((response: IdentityProviderListResponseInterface) => {
                 setPossibleListOfDuplicateIDPs(response?.totalResults
-                    ? response?.identityProviders?.map((eachIdp:StrictIdentityProviderInterface) => eachIdp.name)
+                    ? response?.identityProviders?.map(( eachIdp: StrictIdentityProviderInterface ) => eachIdp.name)
```
Comment (dasuni-30) [inline apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx:255]:
```suggestion
                    ? response?.identityProviders?.map((eachIdp: StrictIdentityProviderInterface) => eachIdp.name)
```

File: apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx [hunk 7]
```diff
@@ -429,6 +430,24 @@ export const AuthenticatorCreateWizardFactory: FunctionComponent<AuthenticatorCr
                     />
                 )
                 : null;
+        case IdentityProviderManagementConstants.IDP_TEMPLATE_IDS.HYPR:
+            return (showWizard && !isEmpty(selectedTemplateWithUniqueName))
+                ? (
```
Comment (dasuni-30) [inline apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx:435]: Check if we can use `&& `instead.

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx [hunk 8]
```diff
@@ -0,0 +1,326 @@
...
+            errors.baseUrl = t("console:develop.features.authenticationProvider.forms.common" +
+                ".requiredErrorMessage");
+        }
+
+        return errors;
+    };
+
+    return (
+        (isIdPListRequestLoading !== undefined && isIdPListRequestLoading === false)
+            ? (
```
Comment (dasuni-30) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx:185]: Check if we can use `&&`

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx [hunk 9]
```diff
@@ -0,0 +1,326 @@
...
+            )
+            : null
+    );
+};
+
+/**
+ * Default props for the Microsoft Authentication Provider Create Wizard Page Component.
+ */
+HyprAuthenticationProviderCreateWizardContent.defaultProps = {
+    "data-testid": "microsoft-idp-create-wizard-page"
```
Comment (dasuni-30) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx:325]: Check if we can use data-componentid

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx [hunk 10]
```diff
@@ -0,0 +1,455 @@
...
+                                    icon={ getIdPIcons().hypr }
+                                    size="mini"
+                                    transparent
+                                    spaced="right"
+                                    data-testid={ `${ testId }-image` }
+                                />
+                                <div className="ml-1">
+                                    { title }
+                                    { subTitle &&
+                                (
```
Comment (dasuni-30) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx:406]: Shall we check formatting here

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx [hunk 11]
```diff
@@ -0,0 +1,326 @@
...
+    return (
+        (isIdPListRequestLoading !== undefined && isIdPListRequestLoading === false)
+            ? (
+                <Wizard
+                    id={ FORM_ID }
+                    initialValues={ { name: template?.idp?.name } }
+                    onSubmit={
+                        (values: HyprAuthenticationProviderCreateWizardFormValuesInterface) => onSubmit(values)
+                    }
+                    triggerSubmit={ (submitFunction: any) => triggerSubmission(submitFunction) }
```
Comment (dasuni-30) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx:192]: Can we avoid the `any` data type and use the correct data type

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx [hunk 12]
```diff
@@ -0,0 +1,324 @@
...
+): ReactElement => {
+
+    const {
+        triggerSubmission,
+        triggerPrevious,
+        changePageNumber,
+        template,
+        setTotalPage,
+        onSubmit,
+        [ "data-componentid" ]: testId
```
Comment (pavinduLakshan) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx:87]:
```suggestion
        [ "data-componentid" ]: componentId
```

Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx]: addressed with 5b48ab3

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx (hunk 1 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx]: addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx (hunk 2 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx]: addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx (hunk 4 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx]: addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)

File: apps/console/src/features/identity-providers/data/identity-provider-templates/templates/hypr/create-wizard-help.tsx (hunk 5 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/data/identity-provider-templates/templates/hypr/create-wizard-help.tsx]: addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx (hunk 3 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx]: addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)

File: apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx (hunk 6 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx]: addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)

File: apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx (hunk 7 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx]: addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx (hunk 8 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx]: addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx (hunk 9 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx]: addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx (hunk 10 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx]: addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx (hunk 11 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx]: addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)

**Review by pavinduLakshan — APPROVED** [review]

File: apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx (hunk 7 above)
Comment (pavinduLakshan) [reply apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx]: Seems like using && for conditional rendering is an anti-pattern.  

Ref: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Comment (brionmario) [reply apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx]: +1. `&&` will result in an `undefined` to be returned as the root element of the component. Which will break React.

https://github.com/wso2/identity-apps/blob/master/docs/write-code/DEVELOPING_REACT_CODE.md#conditional-rendering

Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx]: ack. will check other places as well

**Review by wenodpathirana — APPROVED** [review]

**Review by wenodpathirana — DISMISSED** [review]

File: apps/console/src/features/identity-providers/data/identity-provider-templates/templates/hypr/create-wizard-help.tsx [hunk 13]
```diff
@@ -0,0 +1,185 @@
...
+            </Heading>
+            <p>
+                <Trans
+                    i18nKey={
+                        "console:develop.features.authenticationProvider.templates.hypr" +
+                        ".wizardHelp.baseUrl.description"
+                    }
+                >
+                    Provide the <Code>base URL</Code> of your HYPR server deployment. 
+
```
Comment (pavinduLakshan) [inline apps/console/src/features/identity-providers/data/identity-provider-templates/templates/hypr/create-wizard-help.tsx:151]:
```suggestion
```

File: modules/theme/src/themes/default/assets/images/identity-providers/hypr.svg [hunk 14]
```diff
@@ -0,0 +1,18 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (pavinduLakshan) [inline modules/theme/src/themes/default/assets/images/identity-providers/hypr.svg:1]: Shall we add the license header to this SVG file as well?

Ref: https://github.com/wso2/identity-apps/blob/master/modules/theme/src/themes/default/assets/images/identity-providers/expert.svg?short_path=c0d4482#L1-L17

Comment (Yasasr1) [reply modules/theme/src/themes/default/assets/images/identity-providers/hypr.svg:21]: addressed with [59287e2](https://github.com/wso2/identity-apps/pull/3790/commits/59287e24afb1a5a8a9d6a9eb5e09df6999b3b7e0)

**Review by pavinduLakshan — APPROVED** [review]

**Review by wenodpathirana — APPROVED** [review]

File: apps/console/src/features/identity-providers/data/identity-provider-templates/templates/hypr/hypr.json [hunk 15]
```diff
@@ -0,0 +1,87 @@
...
+        "name": "HYPR",
+        "description": "",
+        "image": "",
+        "isPrimary": false,
+        "isFederationHub": false,
+        "homeRealmIdentifier": "",
+        "certificate": {
+            "certificates": []
+        },
+        "alias": "https://localhost:9444/oauth2/token",
```
Comment (brionmario) [inline apps/console/src/features/identity-providers/data/identity-provider-templates/templates/hypr/hypr.json:20]: Is this expected?

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/identity-providers/data/identity-provider-templates/templates/hypr/hypr.json (hunk 15 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/data/identity-provider-templates/templates/hypr/hypr.json:20]: yes. alias is not used by the hypr idp. adding a default value similar to other idps

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx [hunk 16]
```diff
@@ -0,0 +1,325 @@
...
+    const { t } = useTranslation();
+
+    const [ idpList, setIdPList ] = useState<IdentityProviderListResponseInterface>({});
+    const [ isIdPListRequestLoading, setIdPListRequestLoading ] = useState<boolean>(undefined);
+
+    /**
+     * Loads the identity provider authenticators on initial component load.
+     */
+    useEffect(() => {
+
```
Comment (ThaminduDilshan) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx:99]:
```suggestion
```

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx [hunk 17]
```diff
@@ -0,0 +1,325 @@
...
+                if (idp?.name === value) {
+                    nameExist = true;
+                }
+            });
+        }
+        if (nameExist) {
+            return t("console:develop.features." +
+                "authenticationProvider.forms.generalDetails.name." +
+                "validations.duplicate");
+        }
```
Comment (ThaminduDilshan) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx:143]: Can we simplify this by moving the return statement (line 140) to inside the if blocks (line 135) and removing the local variable nameExist?

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx [hunk 18]
```diff
@@ -0,0 +1,467 @@
...
+                {
+                    "key": "baseUrl",
+                    "value": values.baseUrl.toString()
+                }
+            ];
+
+            if (AppConstants.getClientOrigin()) {
+                if (AppConstants.getAppBasename()) {
+                    identityProvider.image = AppConstants.getClientOrigin() +
+                    "/" + AppConstants.getAppBasename() +
```
Comment (ThaminduDilshan) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx:176]: Check for indentation

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx [hunk 19]
```diff
@@ -0,0 +1,467 @@
...
+            setIsSubmitting(true);
+            createIdentityProvider(identityProvider)
+                .then((response: any) => {
+                    eventPublisher.publish("connections-finish-adding-connection", {
+                        type: componentId
+                    });
+
+                    dispatch(addAlert({
+                        description: t("console:develop.features.authenticationProvider.notifications.addIDP." +
+                        "success.description"),
```
Comment (ThaminduDilshan) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx:203]: Check for indentation. Check other places as well

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx [hunk 20]
```diff
@@ -0,0 +1,467 @@
...
+                        message: t("console:develop.features.authenticationProvider.notifications.addIDP." +
+                        "success.message")
+                    }));
+
+                    // The created resource's id is sent as a location header.
+                    // If that's available, navigate to the edit page.
+                    if (!isEmpty(response.headers.location)) {
+                        const location: any = response.headers.location;
+                        const createdIdpID: any = location.substring(location.lastIndexOf("/") + 1);
+
```
Comment (ThaminduDilshan) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx:214]:
```suggestion
```

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx [hunk 21]
```diff
@@ -0,0 +1,467 @@
...
+                    // Since the location header is not present, trigger callback without the id.
+                    onIDPCreate();
+                })
+                .catch((error: any) => {
+                    const identityAppsError: IdentityAppsError = identityProviderConfig.useNewConnectionsView
+                        ? IdentityProviderManagementConstants.ERROR_CREATE_LIMIT_REACHED
+                        : IdentityProviderManagementConstants.ERROR_CREATE_LIMIT_REACHED_IDP;
+
+                    if (error.response.status === 403 &&
+                    error?.response?.data?.code ===
```
Comment (ThaminduDilshan) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx:229]: Check for indentation

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx [hunk 22]
```diff
@@ -0,0 +1,467 @@
...
+            </>
+        );
+    };
+
+/**
+ * Default props for the Microsoft Authentication Provider Create Wizard.
+ */
+HyprAuthenticationProviderCreateWizard.defaultProps = {
+    currentStep: 1,
+    "data-componentid": "microsoft-idp",
```
Comment (ThaminduDilshan) [inline apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx:465]: Invalid identifier values

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx (hunk 16 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx:99]: shouldn't we leave a blank line after function signature?

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx (hunk 17 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard-content.tsx:143]: addressed with a new PR

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx (hunk 18 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx:176]: addressed with a new PR

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx (hunk 19 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx:203]: addressed with a new PR

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx (hunk 20 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx:214]: addressed with a new PR

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx (hunk 21 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx:229]: addressed with a new PR

File: apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx (hunk 22 above)
Comment (Yasasr1) [reply apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx:465]: addressed with a new PR

**Pull request merged**